magefile.go

```diff
@@ -565,3 +573,25 @@ func checkEnvVar(ckv *checkEnv) (string, pterm.TableData, error) {
 		return "", nil, nil // Unreachable.
 	}
 }
+
+// ExportPluginDocs runs ansible-doc-extractor to generate a markdown file of the plugin documentation.
+func ExportPluginDocs() error {
```

Comment:
I've tried to organize this file using following structure:
1. Imports
2. Constants
3. Mage targets
4. Helper functions called by mage targets

There is even a header which denotes the beginning of helper functions block.
```go
// ----------------------------------- //
//          Helper Functions           //
// ----------------------------------- //
```

Please either remove that header and we will agree to not use that structure or place this new function above that header :)

Reply:
That's fine by me. I have sidebar that auto sorts so I never noticed. I'll push that later and merge. Cheers.

Reply:
Finally fixed. I thought about it before and would love a linter that did this. Not sure I want to try and keep all my files sorted  it makes navigation manually nicer for sure. 

Be a cool linting project.